plugins/source/heroku/Makefile

```diff
@@ -23,3 +23,7 @@ install-hooks:
 gen-docs:
 	rm -rf ./docs/tables/*
 	go run main.go doc ./docs/tables
+
+.PHONY: lint
+lint:
+	golangci-lint run
```

Comment:
Not sure, but I don't think this works correctly; it would have to point to the golangci-lint config file two directories up: `--config ../../.golangci.yml`